### ISSUE TITLE
Support addition and removal of specific roles

### ISSUE THREAD

**Issue by Lightseypoo** [abraker95/game-leaderboards-bot#1]
AddRole [role]
RemRole [role]

freely assigned roles: [dev, nsfw, speed main, accuracy main, stamina main, osu!mania, Etterna, FFR, Robeats, Quaver]

One important piece of bot logic to note is that only one of [osu!mania, etterna, ffr, robeats, quaver] can be active at one time. so if a user with the quaver role sets the role etterna, the quaver role would also need to be removed.

This can exist front-end only.

**Comment by abraker95** [comment]
I think description is missing user to give/remove the role to

**Comment by Lightseypoo** [comment]
> I think description is missing user to give/remove the role to

no, sorry. I should have clarified better. this is not an administrative command, any user can use this. i also edited the main post to include game roles

**Comment by abraker95** [comment]
**Command syntax**
`>>addrole [required:role]`

**Usage example**
`>>addrole -role accuracy main, stamina main`

**Description:**
Adds roles to user who used the command

**arg1: role**
- Roles to add to user

**Comment by abraker95** [comment]
**Command syntax**
`>>rmvrole [required:role]`

**Usage example**
`>>rmvrole -role accuracy main, stamina main`

**Description:**
Removes roles from user who used the command

**arg1: role**
- Roles to remove from user
